CODEOWNERS

```diff
@@ -7,38 +7,40 @@
 
 # List of components with at least two owners as specified in docs/crate-ownership.
 
+components/calendar/              @Manishearth @sffc
 components/collator/              @hsivonen @echeran
+components/collections/           @echeran @iainireland
 components/datetime/              @zbraniecki @gregtatum @nordzilla
 components/decimal/               @sffc
-components/icu/                   @unicode-org/icu4x-owners
-components/icu4x/                 @unicode-org/icu4x-owners
-components/locale_canonicalizer/  @dminor @zbraniecki
+components/list/                  @robertbastian
 components/locid/                 @zbraniecki @nciric
+components/locid_transform/       @dminor @zbraniecki
 components/normalizer/            @hsivonen @echeran
 components/plurals/               @zbraniecki @sffc
-components/uniset/                @echeran @iainireland
+components/timezone/              @nordzilla
 experimental/bies/                @sffc
-experimental/calendar/            @Manishearth @sffc
-experimental/codepointtrie/       @echeran
-experimental/provider_ppucd/      @echeran
+experimental/compactdecimal/      @eggrobin
+experimental/displaynames/        @sffc
+experimental/relativetime/        @pdogr
```

Comment:
potentially add someone else here if there is someone that makes sense (if possible I'd like there to be two component owners especially for situations where one owner is working on icu4x on the side)

Reply:
Who? One owner is better than nobody.

Reply:
Oh, absolutely, but as long as we're adding it we could potentially add two.

Then again it seems like it's mostly been us reviewing Pawan's work so I'm fine leaving it as just him for now